addOns/spider/src/main/java/org/zaproxy/addon/spider/internal/ui/IrrelevantParametersTableModel.java

```diff
@@ -35,7 +35,7 @@ class IrrelevantParametersTableModel
     private static final String[] COLUMN_NAMES = {
         Constant.messages.getString("spider.options.irrelevantparameter.table.header.enabled"),
         Constant.messages.getString("spider.options.irrelevantparameter.table.header.regex"),
-        Constant.messages.getString("spider.options.irrelevantparameter.table.header.value")
+        Constant.messages.getString("spider.options.irrelevantparameter.table.header.name")
```

Comment:
Better correct everywhere.

Reply:
It doesn't appear in the API (that I see, having searched "irr" for Irrelevant).
It's already correct in the Add and Modify dialogs.
It doesn't seem to be anywhere else in the resource file:
![image](https://github.com/user-attachments/assets/995ecf81-436f-4533-8a8e-c8d6caf72654)

Reply:
`IrrelevantParametersTableModel` line 77 `getValue()`

Reply:
Done

If you'd rather the method was actually removed just let me know and I'll add it to the japicmp exclusions.

Reply:
It's an internal class, there's no need to deprecate. The constant should be renamed too.

Reply:
`DialogAddIrrelevantParameter` local variable in `performAction`.